### ISSUE TITLE
[Bug]: notebook images page (AKA CNBi) goes blank

### ISSUE THREAD

**Issue by codificat** [thoth-station/odh-dashboard#12]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I hit a situation where the `/notebookImages` page suddenly goes blank.

Firefox's console shows:

```
TypeError: a is undefined
    Pe CREResourcesTable.tsx:170
    Pe CREResourcesTable.tsx:166
    React 7
    x scheduler.production.min.js:13
    A scheduler.production.min.js:14
react-dom.production.min.js:189:29
Uncaught TypeError: a is undefined
    Pe CREResourcesTable.tsx:170
    Pe CREResourcesTable.tsx:166
    React 7
    x scheduler.production.min.js:13
    A scheduler.production.min.js:14
CREResourcesTable.tsx:170:6
```

### Expected Behavior

No TypeError / page going blank

### Steps To Reproduce

I am not fully sure of what triggers this, but I have hit it a couple of times after creating 2 or more Custom Runtime Environments of type "import image".

I'm using Firefox.

I just checked the same page from Chrome (Chromium) and I do not get the error.

### Open Data Hub Version

```yml
odh-dashboard version cnbi-v0.2.0
```

**Comment by codificat** [comment]
seeing this with Firefox 107.0 on Fedora 36, not seeing it on Chromium 107.0.5304.110

**Comment by Gkrumbach07** [comment]
This error is coming form the sorting in the CRE table on the home page. Upon further investigation, the sorting is not working properly.

**Comment by Gkrumbach07** [comment]
this issue originated form the backend. When etching the CREDetails, the imagestream overwrites any duplicate params with its own. For some reason the image stream was giving undefined for param `name`. which resulted in errors when sorting on param `name`.

The fix was to have the CRE resource overwrite the imagestream

**Comment by codificat** [comment]
This was fixed in #13 

Thanks!

/close
